lib/3scale/api/client.rb

```diff
@@ -297,6 +297,37 @@ def find_account(criteria)
         extract(entity: 'account', from: response)
       end
 
+      # @api public
+      # @return [Array<Hash>]
+      def list_activedocs
+        response = http_client.get('/admin/api/active_docs')
+        extract(collection: 'api_docs', entity: 'api_doc', from: response)
+      end
+
+      # @api public
+      # @param [Hash] attributes ActiveDocs attributes
+      # @return [Hash]
+      def create_activedocs(attributes)
+        response = http_client.post('/admin/api/active_docs', body: attributes)
+        extract(entity: 'api_doc', from: response)
+      end
+
+      # @api public
+      # @param [Fixnum] id ActiveDocs ID
+      # @param [Hash] attributes ActiveDocs attributes
+      # @return [Hash]
+      def update_activedocs(id, attributes)
+        response = http_client.put("/admin/api/active_docs/#{id}", body: attributes)
+        extract(entity: 'api_doc', from: response)
+      end
+
+      # @api public
+      # @return [Hash]
+      def delete_activedocs(id)
+        http_client.delete("/admin/api/active_docs/#{id}")
```

Comment:
I know for what I've been told in #47 that we don't check parameters here intentionally, but it makes me a little bit uncomfortable in the case of a deletion. What would happen if "id" is empty?
"/admin/api/active_docs/" would be called. What would be the outcome in that case?